internal/identifier/convert_test.go

```diff
@@ -0,0 +1,31 @@
+package identifier_test
+
+import (
+	. "github.com/dogmatiq/imbue/internal/identifier"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("func ToScreamingSnakeCase()", func() {
```

Comment:
Does this needed a test for a single word? I did try it in the Go playground and couldn't break it though.

Reply:
Yep I'll add that test :+1:

Reply:
Done